tests/09-ipv6/js/ping-test.js

```diff
@@ -0,0 +1,45 @@
+TIMEOUT(10000, log.testFailed());
+
+dst_ip_addr = "fe80::202:2:2:2";
```

Comment:
Oops..., this script uses only the link-local address of node-2. I'll fix this soon.